admin/src/pages/View/index.js

```diff
@@ -150,7 +150,7 @@ const View = () => {
     else
       return [...subItems, ...acc];
   }, []);
-  const filteredList = !isSearchEmpty ? filteredListFactory(changedActiveNavigation.items, (item) => item?.title.includes(searchValue)) : [];
+  const filteredList = !isSearchEmpty ? filteredListFactory(changedActiveNavigation.items, (item) => (item?.title || '').toLowerCase().includes(searchValue.toLowerCase())) : [];
```

Comment:
```suggestion
  const filteredList = !isSearchEmpty ? filteredListFactory(changedActiveNavigation.items, (item) => (item?.title || '').toLowerCase().includes(normalisedSearchValue)) : [];
```
`searchValue` should be mapped to lower-case version once, not for every check.